binding/binding_test.go

```diff
@@ -39,8 +39,8 @@ type FooStructUseNumber struct {
 }
 
 type FooBarStructForTimeType struct {
-	TimeFoo time.Time `form:"time_foo" time_format:"2006-01-02" time_utc:"1" time_location:"Asia/Chongqing"`
-	TimeBar time.Time `form:"time_bar" time_format:"2006-01-02" time_utc:"1"`
+	TimeFoo time.Time  `form:"time_foo" time_format:"2006-01-02" time_utc:"1" time_location:"Asia/Chongqing"`
```

Comment:
please remove one space `time.Time` after

Reply:
I put space to follow code formatting guidelines, see https://golang.org/doc/effective_go.html#formatting. Do you want to remove it?

Reply:
Ok you are right